libraries/mbed/targets/cmsis/TARGET_STM/TARGET_STM32L4/TARGET_DISCO_L476VG/TOOLCHAIN_ARM_MICRO/startup_stm32l476xx.s

```diff
@@ -39,18 +39,8 @@
 ; 
 ;*******************************************************************************
 
-; Amount of memory (in bytes) allocated for Stack
-; Tailor this value to your application needs
-; <h> Stack Configuration
-;   <o> Stack Size (in Bytes) <0x0-0xFFFFFFFF:8>
-; </h>
-
-Stack_Size      EQU     0x00000400
```

Comment:
MY question could be answered by describing the change in the commit. I can see here removal of Stack_size, Stack_mem, adjusting heap size and changing heap_limit calculation. PLease add more details for this changeset - why, how..